content/docs/use-cases/ci-and-ml.md

```diff
@@ -0,0 +1,171 @@
+# CI for ML
+
+## Motivation
+
+ML engineers and data scientists face many unique challenges. Some of these
+challenges can be solved by adopting first-citizen concepts from traditional
+software engineering and development. Probably the most common adoption is
+versioning control. Some other, unfortunately less common, could be unit testing
+data processing steps of a pipeline. Here is a non exhaustive list of a few
+other unique challenges:
+
+- Automatically re-train models upon changes.
+- Compare (and report) performance of different versions of a model.
```

Comment:
> Compare (and report) performance...

Maybe the challenge isn't that directly but rather doing it systematically or automatically?

Reply:
I'll add "systematically".

Reply:
Cool